### ISSUE TITLE
suggested documentation additions

### ISSUE THREAD

**Issue by RichMorin** [sdg-mit/gitless#193]

**Comment by RichMorin** [comment]
Somehow this issue got pushed before I had a chance to write anything.  Owell...

Anyway, I'd like to suggest augmenting the documentation on the landing page (https://gitless.com/) in a couple of ways. First, I'd like to have formal documentation of the options for each sub-command. Second, it should be possible to get information on a sub-command by typing "gl -h foo".

**Comment by docoleman** [comment]
Are we thinking almost man page documentation? That might be out of scope for this ticket :).

It looks like all the commands have documentation on the homepage now, so that might take care of the first point.

It looks like each sub-command has documentation (you can get at it by doing `gl <command> -h`, e.g. `gl track -h`. This is the same way standard git works.

I'm sure there are still improvements we need, but let's discuss.

**Comment by spderosso** [comment]
> Are we thinking almost man page documentation? That might be out of scope for this ticket :).

I agree. Also, while having a man page would be nice it sounds like too much work for very little gain since the docs are online already.

> It looks like each sub-command has documentation (you can get at it by doing gl <command> -h, e.g. gl track -h.

Maybe we should just add that to the beginning of the "Interface" section? Something like:

"A list of Gitless's commands is included below. For documentation on each command do `gl <command> -h` (e.g, `gl track -h`)"
